README.md

```diff
@@ -240,6 +282,52 @@ gpio partition string input.parquet output/ --column mgrs_code --chars 2
 gpio partition string input.parquet output/ --column region --chars 1 --hive
 ```
 
+#### partition h3
+
+Partition a GeoParquet file by H3 hexagonal cells at a specified resolution. Automatically adds H3 column if it doesn't exist.
+
+```
+$ gpio partition h3 --help
+Usage: gpio partition h3 [OPTIONS] INPUT_PARQUET [OUTPUT_FOLDER]
+
+  Partition a GeoParquet file by H3 cells at specified resolution.
+
+  Creates separate GeoParquet files based on H3 cell prefixes at the
+  specified resolution. If the H3 column doesn't exist, it will be
+  automatically added before partitioning.
+
+  Use --preview to see what partitions would be created without actually
+  creating files.
+
+Options:
+  --h3-name TEXT           Name of H3 column to partition by (default:
+                           h3_cell)
```

Comment:
Curious on the choice of h3_cell - have you seen that be widely used, or you just picked a sensible one? 

I ask because I want to 'standardize' it a bit with a vecorel extension - then we can do quick validation of a file advertises that it implements h3 vecorel extension and has the right column name. I think this is totally fine for now, just we may want to change the default in the future to match vecorel (but we will likely be the ones to make that choice)